core/ecschema-editing/src/Editing/MutableCAClass.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { CustomAttributeClass, CustomAttributeContainerType } from "@bentley/ecschema-metadata";
+
+/**
+ * @internal
```

Comment:
Looks like a bunch of these classes are missing from the api.md file.  Are they not imported into the barrel?

Reply:
No, they are not.  These mutable classes were extracted from the source files in ecschema-metadata (ie. Class.ts) and moved here as individual files.  They are used internally for editing purposes.  What would you like me to do here?

Reply:
Let's just leave them out of the barrel. less chance someone will use them.